### ISSUE TITLE
Crawler should skip URLs that use location.ref to redirect pages

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/WSU-Web-Crawler#11]
If a page loads and uses `window.location.href = ` or `top.location.href = ` to redirect a page, the Accessibility Collector will error out as pa11y is not able to follow the redirect. These sites should be marked by the crawler to avoid further scanning.

If the right regex can be used, we can attempt to retrieve the URL the page is redirecting to, but that may not be possible.

**Comment by jeremyfelt** [comment]
Fixed in ab7762d1e73436d817d751e31b0ea6d8c6f9446b
